sync.py

```diff
@@ -199,12 +203,31 @@ def sync_metarepo(clone_dir, name, submodules):
         logging.info('Meta repository %s requires no changes', name)
 
 
-def handle_event(args, payload):
+def handle_events(args, event_queue):
```

Comment:
I think it makes sense to have `handle_events` as a method in the `RequestHandler` class (and then not have to pass around `event_queue` in three different places).